Src/Base/AMReX_DistributionMapping.cpp

```diff
@@ -766,14 +767,24 @@ DistributionMapping::KnapSackDoIt (const std::vector<long>& wgts,
     Vector<Vector<int> > wrkerord;
 
     if (nteams == nprocs) {
-	LeastUsedCPUs(nprocs,ord);
-	wrkerord.resize(nprocs);
-	for (int i = 0; i < nprocs; ++i) {
-	    wrkerord[i].resize(1);
-	    wrkerord[i][0] = 0;
```

Comment:
I believe this is unneeded because `wrkerord` is unused  in the remainder of the function for `nteams == nprocs`